craft_parts/packages/base.py

```diff
@@ -175,7 +177,9 @@ def refresh_build_packages_list(cls) -> None:
         """Refresh the build packages cache."""
 
     @classmethod
-    def install_build_packages(cls, package_names: List[str]) -> List[str]:
+    def install_build_packages(
+        cls, package_names: List[str], list_only: bool = False
+    ) -> List[str]:
         """Install packages on the host system."""
         return []
```

Comment:
this one could also be an abstract method

Reply:
The method in the base class is abstract, this is the concrete implementation in `DummyRepository`.

Reply:
doh, thanks, I missed the collapsed code and thought it was a continuation of what was rendered just above